packages/cli/package.json

```diff
@@ -107,7 +107,7 @@
     "alpha-sort": "2.0.1",
     "ansi-escapes": "3.0.0",
     "ansi-regex": "3.0.0",
-    "arg": "2.0.0",
+    "arg": "5.0.0",
```

Comment:
Can we remove `mri` from the package.json, or is it used somewhere else?

Reply:
@AndyBitz `mri` still used for `vc secrets` because there is a weird corner case where a secrets value starts with `-` and we can't use `arg`. However, we'll probably deprecate and remove secrets soon.